### PR TITLE
Add documentation for ActiveSupport::StringInquirer [ci skip]

### PR DESCRIPTION
**Pull request by kyatul**
Added an instantiation example in documentation for StringInquirer. Previously it consisted of only a use case of it.

### CODE REVIEW

File: activesupport/lib/active_support/string_inquirer.rb [hunk 1]
```diff
@@ -8,6 +8,12 @@ module ActiveSupport
   # you can call this:
   #
   #   Rails.env.production?
+  #
+  # == Instantiating a new StringInquirer by wrapping a string inside it.
+  # vehicle = ActiveSupport::StringInquirer.new('car')
+  #
+  # vehicle.car?   # => true
+  # vehicle.bike?  # => false
```
Comment (kaspth) [inline activesupport/lib/active_support/string_inquirer.rb]: The code should be indented by 2 spaces.

File: activesupport/lib/active_support/string_inquirer.rb [hunk 2]
```diff
@@ -8,6 +8,12 @@ module ActiveSupport
   # you can call this:
   #
   #   Rails.env.production?
+  #
+  # == Instantiating a new StringInquirer by wrapping a string inside it.
+  # vehicle = ActiveSupport::StringInquirer.new('car')
+  #
```
Comment (kaspth) [inline activesupport/lib/active_support/string_inquirer.rb]: Let's swap this empty line and the line that instantiates the inquirer.

File: activesupport/lib/active_support/string_inquirer.rb [hunk 3]
```diff
@@ -8,6 +8,12 @@ module ActiveSupport
   # you can call this:
   #
   #   Rails.env.production?
+  #
+  # == Instantiating a new StringInquirer by wrapping a string inside it.
```
Comment (kaspth) [inline activesupport/lib/active_support/string_inquirer.rb]: Let's just remove the `by wrapping a...`

Comment (kyatul) [inline activesupport/lib/active_support/string_inquirer.rb]: @kaspth I have made the above changes as mentioned by you, and pushed the new commits.

**Comment by kaspth** [comment]
Thanks!

**Pull request merged**